modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -573,19 +575,42 @@ You can use the `ginkgo.focus` flag to run a specific test.
 [id="cnf-performing-end-to-end-tests-running-hwlatdetect"]
 === Running hwlatdetect
 
-To perform the `hwlatdetect`, run the following command:
+The `hwlatdetect` tool is available in the `rt-kernel` package with a regular subscription of Red Hat Enterprise Linux 8.
+
+.Prerequisites:
+
+* You installed the real-time kernel
+* You run the tool runs on bare hardware
+* You are logged into registry.redhat.io with your Customer Portal credentials
+
+.Procedure
+
+. Run the following command:
 
 [source, terminal]
 ----
-$ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20  registry.redhat.io/openshift4/cnf-tests-rhel8:v4.9 /usr/bin/test-run.sh -ginko.focus=”hwladetect”
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e \
+LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf -e \
+LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20  registry.redhat.io/openshift4/cnf-tests-rhel8:v4.9 \
+/usr/bin/test-run.sh -ginkgo.focus="hwlatdetect"
 ----
 
-The above command runs the `hwlatdetect` tool for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs), and the command line displays `SUCCESS!`.
+The command runs the `hwlatdetect` tool for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs), and the command line displays `SUCCESS!`.
+
+[NOTE]
+====
+For valid results, it is recommended to run the test for at least 12 hours.
+====
 
 .Example failure output
```

Comment:
Do we have successful output to highlight key parameters. We are showing a failure here to illustrate key parameters. Maybe we just need some context here, why are we showing failed output ?

Reply:
@kquinn1204 The successful output is there in the docs. When the test is successful, it just outputs `SUCCESS!`
> The command runs the `cyclictest` tool for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs), and the command line displays `SUCCESS!` when this test is completed.

As Martin Sivak explained, this is what happens:
> Internally the output is the same, we just compare the max latency figures with the expected and silence it when all is good

I'll add some clarification